book/website/index.md

```diff
@@ -1,75 +1,124 @@
-That means investing in the socio-technical skills required to work in a team, to build something more significant than any individual could deliver alone.
+Our goal is to provide individuals and organizations in academia, industry, government, and the public sectors the information they need to ensure that their projects are easy to reproduce and reuse. 
+We encourage all stakeholders in data science to utilise _The Turing Way_ to understand their roles and responsibilities for reproducibility.
 
-_The Turing Way_ is:
+**What Awaits You**
 
-* a book
-* a community
-* a global collaboration
+* Collaboration Beyond Boundaries: Engage in a global collaboration that transcends borders and welcomes diverse perspectives.
+* Skill Development: Learn a skills, share a skill that you have and contribute to projects in making a real impact.
+* Learn, Remix, Reuse: Dive into our resources, and feel free to remix and reuse materials (just remember to attribute as per our [License](https://github.com/the-turing-way/the-turing-way/blob/main/LICENSE.md)].
 
-We hope you find the content in the book helpful.
-Everything here is available for free under a [CC-BY licence](https://github.com/the-turing-way/the-turing-way/blob/main/LICENSE.md).
-Please use and re-use whatever you need, for any purpose.
+**Calling Everyone In: Your Participation and Perspectives Matter!**
 
-_The Turing Way_ receives support and funding from [The Alan Turing Institute](https://www.turing.ac.uk/), however, the project is designed to be a global collaboration.
-We have contributions from across the UK, India, Mexico, Australia, the USA, some African countries, and many European countries.
-Chapters have been written, reviewed and curated by members of research institutes and universities, government departments, and industry.
-We are committed to creating a space where people with diverse expertise and experiences can share their knowledge with others, together supporting each other in using data science to improve the world.
+The Turing Way community is a open and collaborative space. 
+It is an ecosystem where all stakeholders of data science come together to make research "too easy not to do." 
+We invest in socio-technical skills, fostering a team spirit to build something greater than any individual effort.
 
 ```{admonition} Translation
 :class: tip
 Call for translators! We’re looking for contributors to help translate this space for everyone! Read more about our translation workflow and how you can contribute to it in the {ref}`translation chapter<ch-translation>`.
 ```
 
-We value the participation of every member of our community and want to ensure that every contributor has an enjoyable and fulfilling experience.
-Accordingly, everyone who participates in _The Turing Way_ project is expected to show respect and courtesy to other community members at all times.
+```{figure} figures/theturingway-chapters.*
+---
+name: theturingway-chapters
```

Comment:
OK, so this is an actual issue as the same figure with the same name got actually merged into the book at a separate page: https://github.com/the-turing-way/the-turing-way/blob/main/book/website/foreword/navigate.md and now there is a clash with the references.

@the-turing-way/infrastructure-working-group - do we have a preferred way to deal with the reuse of figures at multiple locations? The easy workaround is to use a different name, but I wonder how that works with the the figure gallery/index page (or shall we cross that bridge when we finally get there?). 
Also, is there a way to reuse the alt-text and have the exact same thing for both?

Reply:
I'm not sure if there is a way to have the same figure in two places.

If you put the Markdown for the figure in a separate file and included it in two places you would still get the figure name clash.

It will look odd in the figure gallery to have two identical figures though.

Reply:
Ping @agoose77?

Reply:
I think we should just use a different name.
I haven't been keeping up with [the gallery discussions](https://github.com/the-turing-way/the-turing-way/issues/2046) but I'd sort of been assuming that that would load images from the figures folder (or wherever [we end up keeping images in the future](https://github.com/the-turing-way/the-turing-way/issues/3843)) rather than having a script go through the book and see what figures are actually used.

Reply:
For the gallery, I would definitely would love to see both a "Gallery of everything" as a couple of the really cool illustrations are not necessarily made for TTW but for adjacent projects. Maybe this could live in a separate repo.

And I would like to see a Table of Figures, for everything we use in the book. And here all the duplicates should hopefully be found and cleaned up/made self-consistent.

Reply:
(That said, in this PR I agree that we can be pragmatic and use a different name, and to figure out these gallery and figure policies separately.)

Reply:
@sgibson91 oops. Missed the `mention` part here.

@JimMadge in JB2 (`mystmd`), you can embed figures such that the embedded figures do not increment reference counts:

````markdown
:::{figure} image.png
:name: img-1

An image!
:::

See @img-1, or look below:

![](#img-1)

An image appears!
````

In the EB stack, we don't have a way to duplicate a figure without creating effectively a new figure.

Reply:
That's great! Another thing to look forward to in MystMD :smile: